src/components/Whiteboard/ElementBehaviors/Selection/types.ts

```diff
@@ -14,8 +14,18 @@
  * limitations under the License.
  */
 
+import { Elements } from '../../../../state/types';
+
 export type WithSelectionProps = {
   readOnly: boolean;
   active: boolean;
   elementId: string;
 };
+
+export type WithExtendedSelectionProps = {
+  readOnly: boolean;
+  active: boolean;
+  elementId: string;
+  elementIds: string[];
```

Comment:
I would make `elementIds` and `overrides` optional and set a default in the components.
Then we save `elementIds={[]}` in some places.

Plus `elementIds` should eventually be named `activeElementIds` instead.

Reply:
Yup, makes sense